publication/src/main/scala/hmda/publication/reports/aggregate/A42.scala

```diff
@@ -196,7 +238,9 @@ object A42 {
          |    "total": $total
          |}
          |
-       """.stripMargin.parseJson
+       """.stripMargin
+
+      AggregateReportPayload(reportId, fipsCode.toString, report)
```

Comment:
The S3 path depends on the `msa` field in the payload, and since the National reports are being generated with a `-1`, doing something similar to [Disclosure Report B](https://github.com/cfpb/hmda-platform/blob/master/publication/src/main/scala/hmda/publication/reports/disclosure/DisclosureB.scala#L163) might make the file path a little easier to read.

Reply:
Good call. Done in 8c8887807aa0cc6699e1d2886f01bf363eda91e3.